roles/kubernetes/node/tasks/main.yml

```diff
@@ -152,7 +152,7 @@
     - kube-proxy
 
 - name: Cleanup kube-proxy leftovers from node
-  command: "{{ local_release_dir }}/hyperkube kube-proxy --cleanup"
+  command: "{{ docker_bin_dir }}/docker exec -it $(docker ps -af name=k8s_kube-proxy* -q) kube-proxy --cleanup --config /var/lib/kube-proxy/config.conf"
```

Comment:
I don't have any setup using kube-router network plugin to test that, hope the CI can pass a test for this

Reply:
I would trigger kube-router CI job to have it tested

Reply:
In the meantime, I had some thoughts about that. I'm not very found of the use of docker for that especially as it won't work with other container engines.
I guess we can live with having both kubelet and hyperkube binaries in `bin_dir`

Reply:
That's a good point, I reverted the commit that was using the docker command.